process/templates/SIG_NAME_sandbox_stage.md

```diff
@@ -0,0 +1,15 @@
+## Creation of a new Special Interest Group (SIG) at Sandbox stage
+
+### Proposed focus, intent, goals, and/or deliverables
+
+  * Link to relevant documentation.
+
+### List of interested individuals
+
+The initial membership should be at least 3 individuals from 2 different organizations
```

Comment:
```suggestion
The initial membership should be at least three individuals from at least two distinct organizations
```

Notes:
1. There's often a style rule in favor of writing out numbers `1`...`9`.
2. I can't quite explain why I think `distinct` is better than `different`, but I think I'm right...
3. Can people be unaffiliated as opposed to be from organizations? (If so, can all interested individuals be unaffiliated?)